### PR TITLE
IN-251 Added information on Name Range Comment

### PR DESCRIPTION
**Pull request by franciscoatsantos**

### CODE REVIEW

File: src/ValiAddon.bas [hunk 1]
```diff
@@ -297,30 +309,31 @@ Sub PushValis()
             Response = ValiAPI("rest/valis/" & ValiID, "PATCH", Data)
             'MsgBox (Response)
             Set JSONResponse = JsonConverter.ParseJson(Response)
+            MsgBox JSONResponse("name") & JSONResponse("name") & JSONResponse("project")
```
Comment (antopemo) [inline src/ValiAddon.bas:312]: Is this a debugging leftover?

**Review by antopemo — CHANGES_REQUESTED** [review]

File: src/ValiAddon.bas (hunk 1 above)
Comment (franciscoatsantos) [reply src/ValiAddon.bas:312]: Might be, what is your take on it? @cauenapier

**Comment by cauenapier** [comment]
Ops, I closed a discussion by mistake and don't find where to reopen it.
But yes, that MsgBox was a debugging left over.

**Review by cauenapier — CHANGES_REQUESTED** [review]
LGT, just that variable name that I think it might be better to rename. Just for improving readability.

File: src/ValiAddon.bas (hunk 1 above)
Comment (cauenapier) [reply src/ValiAddon.bas:312]: That MsgBox was a debugging left over.

**Pull request merged**